ansible/roles/master/templates/haproxy.cfg.j2

```diff
@@ -0,0 +1,73 @@
+kip to content
```

Comment:
I would put this here: ansible/roles/haproxy/templates/haproxy.cfg.j2

And add something like the following to https://github.com/kubernetes/contrib/blob/master/ansible/cluster.yml#L53-L59:

```
    - { role: haproxy, when: groups[masters] | length > 1}
```

**Note**: Take the same approach for keepalived